src/pages/kimdaseul/Login/Login.js

```diff
@@ -1,13 +1,88 @@
 import React from 'react';
+import { withRouter } from 'react-router-dom';
+import './Login.scss';
 
 class LoginSeul extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+      pw: '',
+      opacity: false,
+      disabled: true,
+    };
+  }
+  handleInput = e => {
+    const { value, name } = e.target;
+    this.setState({
+      [name]: value,
+    });
+    console.log(value);
+    const isValid = this.state.id.includes('@') && this.state.pw.length >= 5;
+    this.setState({
+      opacity: isValid,
+      disabled: !isValid,
+    });
+  };
+
+  goToMain = () => {
+    this.props.history.push('/mainseul');
+  };
+
+  signupLogin = () => {
+    fetch('http://10.58.3.99:8000/users/login', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.id,
+        password: this.state.pw,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => {
+        if (result.access_token) {
+          localStorage.setItem('token', result.access_token);
+          this.props.history.push('/mainseul');
+        } else {
+          alert('다시 입력');
+        }
+      });
+  };
```

Comment:
- 구조분해할당 사용해 주시면 더 좋을 것 같습니다. 
```suggestion
  const { id, pw } = this.state;
  signupLogin = () => {
    fetch('http://10.58.3.99:8000/users/login', {
      method: 'POST',
      body: JSON.stringify({
        email: id,
        password: pw,
      }),
    })
      .then(response => response.json())
      .then(result => {
        if (result.access_token) {
          localStorage.setItem('token', result.access_token);
          this.props.history.push('/mainseul');
        } else {
          alert('다시 입력');
        }
      });
  };
```

Reply:
수정 완료했습니다!